packages/block-editor/README.md

```diff
@@ -613,6 +625,12 @@ _Related_
 
 -   <https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/skip-to-selected-block/README.md>
 
+### START_OF_SELECTED_AREA
+
+A robust way to retain selection position through various
```

Comment:
I'm not sure these two should be exposed to the public API as it seems it's just implementation details. What do you think?

Reply:
I think it's okay to just use them as before without making them utils. Code duplication will be minimal and no future problems are guaranteed. Even if `experimental` there are [still big discussions](https://github.com/WordPress/gutenberg/issues/40316) about what should be considered public or not.

Reply:
@ntsekouras thanks, that makes sense to me. I've reverted the `experimental` commit, and changed the new utils to no longer be publicly exposed, but rather to be consumed externally. Let me know what you think.